packages/p2p/src/components/buy-sell/buy-sell-form.jsx

```diff
@@ -1,15 +1,53 @@
+import classNames from 'classnames';
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Formik, Field, Form } from 'formik';
-import { Input } from '@deriv/components';
-import { formatMoney, getDecimalPlaces, getRoundedNumber, getFormattedText, useIsMounted } from '@deriv/shared';
+import { Input, Text } from '@deriv/components';
+import {
+    formatMoney,
+    getDecimalPlaces,
+    getFormattedText,
+    getRoundedNumber,
+    isDesktop,
+    isMobile,
+    useIsMounted,
+} from '@deriv/shared';
 import { localize, Localize } from 'Components/i18next';
 import { countDecimalPlaces } from 'Utils/string';
 import { requestWS } from 'Utils/websocket';
 import { textValidator, lengthValidator } from 'Utils/validations';
 import { buy_sell } from '../../constants/buy-sell';
 
-const BuySellForm = ({ advert, handleClose, handleConfirm, setErrorMessage, setIsSubmitDisabled, setSubmitForm }) => {
+const BuySellFormReceiveAmount = ({ is_sell_advert, receive_amount, local_currency }) => (
```

Comment:
Is this the same as the ConfirmModal in  `packages/p2p/src/components/order-details/order-details-footer.jsx` with the text?